tests/unit/Query.test.ts

```diff
@@ -31,4 +32,16 @@ describe('Query', () => {
     deepStrictEqual(query.terms, new Set(['officially', 'supported']));
     deepStrictEqual(query.phrases, ['officially supported']);
   });
+
+  it('should look for the phrase as a compound should with a boost', () => {
```

Comment:
Could this test name be reworded? It took me a bit of time to understand what it means 😭 the "should" threw me off lol

Reply:
good point! updated